probfit/plotting.py

```diff
@@ -1,7 +1,8 @@
 # -*- coding: utf-8 -*-
-# Plotting is on python since this will make it much easier to debug and adjsut
-# no need to recompile everytime i change graph color....
-# needs a serious refactor
+#Plotting is on python since this will make it much easier to debug and adjust
```

Comment:
Removing a space between the # and the text is not making the comment more readable, please revert.

Reply:
Apologies, made a mistake at the rebase